pytest-saas/exasol/pytest_saas/__init__.py

```diff
@@ -33,6 +33,16 @@ def pytest_addoption(parser):
             pytest plugin for exasol-saas-api will include this short tag into
             the names of created database instances.""",
     )
+    parser.addoption(
+        "--idle-time",
+        action="store",
+        default="2",
```

Comment:
This mean we could have a minimum idle time of 1h. Couldn't there be cases where we want to have a shorter idle-time, maybe 30min or so?

Reply:
Firstly, it's a float, 30 min is 0.5.
Secondly, I don't see cases when we need to set it to a shot period. The database fixture is supposed to shut down the database as soon as all tests are done. Unless we want to keep the database to investigate the effect of our tests. In that case, we probably want to give ourselves enough time for investigation and then shut down the db manually.

Reply:
I am wondering about whether we should rename the CLI option.
I think `--idle-time` is very unspecific, it should say something about the semantics of the idle duration:
* which component is affected, in this case is clear I think: the SaaS databse instance 
* what will happen after the timeout, i.e. an autostop 
* especially the unit should be clear, AFAIK @ahsimb chose hours which also deviates from the unit _minutes_ in the original SaaS API.

In [saas-api-python/.../__init__.py](https://github.com/exasol/saas-api-python/blob/main/exasol/saas/client/__init__.py) the constant is called `AUTOSTOP_DEFAULT_IDLE_TIME` with datatype `timedelta`.